### PR TITLE
feat: base map schedule page

### LINKED ISSUE #2530: Query Sheets for Base Map Update Schedule
Replace the google sheets embed with a build-time query for the sheet.

https://gis.utah.gov/developer/base-maps/basemap-maintenance-schedule/

### PR DESCRIPTION
**Pull request by stdavis**
Closes #2530

Deploy preview page: https://deploy-preview-2615--gis-utah.netlify.app/documentation/discover/maintenance-schedule/

### CODE REVIEW

**Comment by steveoh** [comment]
Awesome! Any chance we can pull and add those specific update values for the base map pages I started?

File: src/utilities/base-maps-dates.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+let client;
+if (import.meta.env.PROD) {
+  console.log('using ci credentials');
+  client = auth.fromJSON(JSON.parse(import.meta.env.GOOGLE_PRIVATE_KEY));
+  client.scopes = scopes;
+} else {
+  client = new GoogleAuth({
+    scopes,
+  });
```
Comment (steveoh) [inline src/utilities/base-maps-dates.ts:16]: **question**: in dev, should we use fake data to avoid rate limiting?

**Review by jacobdadams — APPROVED** [review]
I like it. I don't remember my initial thoughts on the concept, but this solution seems really concise.

File: src/utilities/base-maps-dates.ts (hunk 1 above)
Comment (stdavis) [reply src/utilities/base-maps-dates.ts:16]: Since it's only one call to get the range of cells, I wasn't worried about it.

**Pull request merged**